core/utils.py

```diff
@@ -366,6 +366,14 @@ def validate_json_schema(schema):
         ]
 
 
+def json_serialize_value(value):
+    try:
+        json.dumps(value)
```

Comment:
should not be `value = json.dumps(value)` ?

Reply:
> should not be `value = json.dumps(value)` ?

Because we don't want to double serialize a value. `json.dumps` here is just testing if the value is json serializable. See the original implementation from tasks_management module this implementation is based upon:

```
non_serializable_types = (
    uuid.UUID,
    datetime.date,
    AdDate,
    AdDatetime,
    decimal.Decimal,
)

def serialize_value(value):
    return str(value) if any(isinstance(value, t) for t in non_serializable_types) else value
```

Reply:
perhaps a better function name would be `to_json_safe_value` or `json_safe_value`?

Reply:
`to_json_safe_value` sounds good to me.

Reply:
I've renamed the function as suggested.